enhancements/cluster-logging/gcp-external-account-auth.md

```diff
@@ -0,0 +1,201 @@
+---
+title: gcp-external-account-auth
+authors:
+  - "@cahartma"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@Clee2691"
+  - "@JoaoBraveCoding"
+  - "@xperimental"
+approvers:
+  - "@jcantrill"
+  - "@alanconway"
+api-approvers: 
+  - "@jcantrill"
+creation-date: 2025-02-27
+last-updated: 2024-02-27
+tracking-link:
+  - "https://issues.redhat.com/browse/LOG-3577"
+see-also:
+  - "https://issues.redhat.com/browse/LOG-3275"
+  - "https://issues.redhat.com/browse/LOG-6762"
+replaces: []
+superseded-by: []
+---
+
+# GCP Workload Identity Federation (WIF) Support in OpenShift Logging
+
+This enhancement enables secure, short-lived authentication for GCP services in OpenShift Logging using Workload Identity Federation
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement adds support for Workload Identity Federation (WIF) to the existing GCP 
```

Comment:
This enhancement adds support to GCP authentication for Workload Identity Federation (WIF) to the `ClusterLogForwarder` API , specifically for Vector.

Reply:
While the implementation details are mainly related to vector, this enhancement is must reflect the ClusterLogForwarder and a "collector".  We need to be aware that at some point the collector impl may change but the API will live on.  The API is the primary bit of interest here and should reflect something that is abstracted from the actual collector choice

Reply:
I'm confused.    The enhancement we need is to authenticate via another file type, but that is all abstracted from the CLF by our collector.   Our operator now needs to parse the secret and conditionally project a token.   The rest is all vector and would be the same for any other collector.

Reply:
Given our arch meeting and conversation, maybe the following is more in line:

```
This enhancement extends GCP authentication for Workload Identity Federation (WIF) to support external account authentication , specifically for Vector.
```